src/components/InputBook.js

```diff
@@ -36,28 +37,31 @@ const InputBook = () => {
   };
   return (
     <>
-      <h3>ADD NEW BOOK</h3>
-      <form action="submit" className="submit">
-        <input
-          type="text"
-          className="title"
-          placeholder="Book title"
-          value={data.title}
-          name="title"
-          onChange={handleChange}
-        />
-        <input
-          type="text"
-          className="author"
-          placeholder="Author"
-          name="author"
-          value={data.author}
-          onChange={handleChange}
-        />
-        <button type="button" className="addBook" onClick={handleSubmit}>
-          Add Book
-        </button>
-      </form>
+      <hr className="form-hr" />
+      <div className="add-book">
+        <h3>ADD NEW BOOK</h3>
```

Comment:
Kindly use the same fonts and colors used in the template the title and divider looks diffrent here, also increase left marging of your component 

![image (18)](https://github.com/ragangithub/bookstore-react/assets/59707859/4a0e917f-f739-448f-9021-d5c5d7fe63ba)
![image (23)](https://github.com/ragangithub/bookstore-react/assets/59707859/b5c047db-e2ee-4767-acf0-11640e8b02ce)